src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends FigCol {
```

Comment:
Don't use verbs for class/interface names. The name 'FigCol' suggests it might be a method due to the use of verbs. Consider renaming it to something more indicative of a class or an interface, such as 'ColoredFigure' or similar.